src/scanners/css.js

```diff
@@ -31,25 +31,23 @@ export default class CSSScanner extends BaseScanner {
     }
 
     var file = cssCode.position.source;
-    var startLine = cssCode.position.start.line;
-    var startColumn = cssCode.position.start.column;
+    var options = Object.assign(this.options, {
```

Comment:
This is mutating an instance level options object each time - I'm not too sure about that.